### ISSUE TITLE
WebsitePath Error! after installation

### ISSUE THREAD

**Issue by yarnball** [lincanbin/Carbon-Forum#195]
Hi,

This package looks amazing. 

However, I keep getting the error "WebsitePath Error!" after I enter my server details. The password/db name etc is all correct.

I have tried using it on both my local system (using MAMP) and on my live PHP web sever. Both say they install, but then I keep getting the "WebsitePath Error!".

I have tried visting URL.com/register, and the root domain. /register gives a 404 error. The home/root domain says "WebsitePath Error!".

Any suggestions?

**Comment by lincanbin** [comment]
The program is installed in the sub directory?

**Comment by yarnball** [comment]
No, root directory of the folder (eg www.website.com)

**Comment by lincanbin** [comment]
![qq 20160706111305](https://cloud.githubusercontent.com/assets/5785188/16606445/a293cca8-436a-11e6-81cc-c6858f06c85d.png)
What's the value of ```WebsitePath``` in the database?

**Comment by yarnball** [comment]
Ahh- it says "WebsitePath/carbon". So that means its in the wrong folder?

**Comment by lincanbin** [comment]
Let its value be empty.

**Comment by lincanbin** [comment]
If you move the path of the program, then you need to execute ```/update```.

**Comment by yarnball** [comment]
Ok, I executed update. So this will only work if it is on the root directory (ie NOT in a folder)? Because I'm using MAMP, it won't let me use the root directory

**Comment by lincanbin** [comment]
https://github.com/lincanbin/Carbon-Forum/commit/7315a864be2cf4289bdd9557adf1943b709a313f
No, that's a Bug.

**Comment by yarnball** [comment]
I have just re-installed the latest version (master- 7315a86) but I still have the same problem.

Here's what I did:

1. Cloned into directory (of MAMP) htdocs/carbon
2. Opened localhost:8888/carbon
3. Put in the correct db details
4. It said "installation successful"
5. I clicked "点我马上注册管理员账号
The first registered users will become administrators." and it says "Error: 404 NOT FOUND"

6. I went to my PHP admin and removed "/carbon"  from carbon_config>WebsitePath

What have I done wrong?
